io/iommu/amd/iommu_v2pgmode_test.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2023 Advanced Micro Devices, Inc.
+# Author: Dheeraj Kumar Srivastava <dheerajkumar.srivastava@amd.com>
+
+"""
+Validate 5 level page table support in v2 iommu page table mode
+"""
+
+from avocado import Test
+from avocado import skipUnless
+from avocado.utils import cpu, genio, process, linux_modules
+
+
+def get_va_bits():
+    '''
+    Check for VA address bit size in /proc/cpuinfo
+    '''
+    cpu_info = genio.read_file("/proc/cpuinfo")
+    for line in cpu_info.splitlines():
+        if "address sizes" in line:
+            return line.split()[-3].strip()
+    return ""
```

Comment:
can we push this as a utility  in avocado

Reply:
Sure i will send patch for this util to avocado.

Reply:
Done. I have updated the patch. Please review.